### PR TITLE
fix types for "ignore" in FindPathOpts

### PR DESCRIPTION
**Pull request by lmik**
also fixes MoveToOpts

### CODE REVIEW

**Comment by lmik** [comment]
woops, missed that other commit, i guess its okay too 😅

File: dist/game/path-finder.d.ts [hunk 1]
```diff
@@ -142,29 +152,20 @@ declare module "game/path-finder" {
...
     // // /**
     // //  * Draw a line along the creep’s path using `RoomVisual.poly`. You can provide either an empty object or custom style parameters.
     // //  */
     // // visualizePathStyle?: PolyStyle;
-
-    /**
-     * An array of the room's objects or RoomPosition objects which should be treated as obstacles during the search. This option cannot
-     * be used when the new PathFinder is enabled (use costCallback option instead).
-     */
-    ignore?: any[] | RoomPosition[];
```
Comment (thmsndk) [inline dist/game/path-finder.d.ts:167]: Is this in fact not present on MoveOpts? I seem to recall it being used multiple places?

**Review by thmsndk — APPROVED** [review]
I think the additional ignore was to get a different intellisense comment on it?

**Comment by lmik** [comment]
no need for a different comment, as moveTo() just passes the opts to findPath (as in, its the same)

`opts	object	An object with additional options that are passed to /game/utils::findPath)
`

anyways, just did some testing and its RoomPosition[], not the constructors, my bad.

**Pull request merged**